lib/configure.js

```diff
@@ -321,7 +320,7 @@ module.exports = function() {
 
     function travisAwaitSyncing() {
       process.stdout.write('Waiting for Travis to finish syncing…\n');
-      return new Promise(function(resolve, reject) { setTimeout(resolve, 5000) })
+      return new Promise(function(resolve, reject) { setTimeout(resolve, 5000); })
```

Comment:
I disagree with `semi: 'always'` for single-line single-statement function expressions like this one, which are akin to lambdas, as noted by [Arrow Functions - Shorter functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Shorter_functions). I find the semicolonless variant more readable. But there's no way to configure ESLint to accommodate this preference, and I'd much rather enforce semicolons `always` than `never`, so let it be!